### ISSUE TITLE
Platform support (Node specifically)

### ISSUE THREAD

**Issue by dan-gamble** [RisingStack/react-easy-state#164]
In the README it mentions `node 6+` support but in this commit https://github.com/RisingStack/react-easy-state/commit/d4cf898bd54ed505e88bde28142cbd8bd76d2e19 it was changed to 10+.

Unfortunately, I'm temporarily stuck on a build system that needs 9.x. Was this a mistake and can be changed back to 6.x or should the README be updated to reflect 10.x?

I'll use a version from before that commit, for now, as I really want to give this library a try :)

**Comment by peteyycz** [comment]
If you're using the prebuilt version a.k.a. the one on NPM, which only contains built files you should be fine using any node version for your build system. :) But if you run into any issues please let us know and we'll try to answer. 👍 That node version is only valid if you'd like to build the library yourself after cloning the repository

**Comment by dan-gamble** [comment]
Hey Peter,

This is what I'm doing at the moment:
```bash
project on  branch [$] is 📦 v1.0.0-alpha.1 via ⬢ v9.11.2
➜ yarn add @risingstack/react-easy-state
yarn add v1.17.3
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @risingstack/react-easy-state@6.2.0: The engine "node" is incompatible with this module. Expected version ">=10.0.0". Got "9.11.2"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

In the meantime, I'm using the old `react-easy-state@6.1.3`

If there is any other info that would help, let me know. 👍

**Comment by solkimicreb** [comment]
Yeah adding Node 10+ into package.json engines was a mistake from our side. I has no idea yarn takes it so seriously ):

To clarify:
- Node 10+ is needed to run some development pipelines.
- Node 6+ is to run easy state inside Node (for server side rendering)
- No Node is required if it runs in the browser (obviously)

I released v6.2.1 with the engine req rolled back to Node 6+. Can you check if the new release fixes your issue?

**Comment by dan-gamble** [comment]
Hey Miklos,

That's fixed it, thank you!

I've been playing around with this today and it's amazing what kind of doors this opens.

I love that I can use an array of classes 😍.

Keep up the good work guys 👏
